src/assets/scripts/base/PositionModel.js

```diff
@@ -238,15 +204,28 @@ export default class PositionModel {
     }
 
     /**
-     * Fascade for `adjustForMagneticNorth`
-     *
-     * Adjusts x & y coordinates from true north to magnetic north
-     *
+     * Checks whether or not this `PositionModel` has a reference PositionModel
```

Comment:
👍 for the comments! (be sure to remember the bat ticks on the second `PositionModel` reference though)